common/openapi.yml

```diff
@@ -778,7 +778,6 @@ components:
           type: array
           items:
             type: string
-            format: uuid
```

Comment:
Just out of curiosity, any specific reason to remove this?

Reply:
To keep things concise, rather than try and return two different map types, I opted to change this back to a string, so I could return a Map<String List<String>> but this may change looking through more of the PR comments.

Reply:
With the latest push, this is now back to being a uuid